### PR TITLE
Adds Platform to Braven Dev Network

### PR DESCRIPTION
**Pull request by StetsenTech**
**Ticket(s)**:
https://app.asana.com/0/1133543009734854/1143501603378304/f

**Summary**
- Adds Platform to the Braven Dev Network so that it can resolve names from other services

### CODE REVIEW

File: config/secrets.yml [hunk 1]
```diff
@@ -24,12 +26,14 @@ staging:
   mailer_from_email: <%= ENV['MAILER_FROM_EMAIL'] %>
   devise_secret_key: <%= ENV['DEVISE_SECRET_KEY'] %>
   devise_pepper: <%= ENV['DEVISE_PEPPER'] %>
-  
+  sso_url: <%= ENV['SSO_URL'] %>
+
```
Comment (rshipp) [inline config/secrets.yml]: sso_url was already in there for all of these

File: docker-compose.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-version: '3'
+version: "3.5"
```
Comment (rshipp) [inline docker-compose.yml]: is this the version of docker-compose, or something else? just curious

Comment (StetsenTech) [reply docker-compose.yml]: This is the version of docker-compose we use in other containers. It's the reason we are able to name networks.
https://docs.docker.com/compose/compose-file/compose-versioning/#version-35

File: config/secrets.yml (hunk 1 above)
Comment (StetsenTech) [reply config/secrets.yml]: Ah for some reason it didn't show in mine....removing.

**Comment by StetsenTech** [comment]
Removed extra `sso_url`s. Not sure why it was missing when I was developing...

**Pull request merged**